src/collar/oc_rlvsys.lsl

```diff
@@ -620,7 +630,31 @@ state active
                 llMessageLinked(LINK_SET, TIMEOUT_REGISTER, "30", "recheck_lock");
                 llMessageLinked(LINK_SET, RLV_REFRESH, "","");
                 llMessageLinked(LINK_SET, LM_SETTING_REQUEST, "ALL","");
-            } 
+            } else if(iNum == RLV_CMD_OVERRIDE){
+                //New feature! RLV_CMD_OVERRIDE is designed to allow one-shot (force) commands to override current restrictions. This is done by sending the behavior(s) to send as a comma separated list, followed by a ~ and then the restrictions to temporarily lift as another comma separate list as as sMsg. EXAMPLE: llMessageLinked(LINK_THIS,RLV_CMD_OVERRIDE,"unsit~unsit","") will lift the current unsit restriction if present, issue an @unsit=force, then restore the unsit restriction if it was previously present. These commands should ONLY be sent if the issuer has OWNER auth.
+                list lCommands=llParseString2List(llList2String(llParseString2List(sStr,["~"],[]),0),[","],[]);
+                list lOverrides=llParseString2List(llList2String(llParseString2List(sStr,["~"],[]),1),[","],[]);
+                integer iLen=llGetListLength(lOverrides);
+                while(iLen--) {
+                    if(llListFindList(g_lBaked,llList2List(lOverrides,iLen,iLen))==-1)
+                    {
+                        lOverrides=llDeleteSubList(lOverrides,iLen,iLen);
+                    }
+                    else llOwnerSay("@"+llList2String(lOverrides,iLen)+"=y");
+                }
+                llSleep(0.5); 

```

Comment:
What are we waiting for here?

Reply:
This is between temporarily releasing restrictions that will block a one-shot RLV command and issuing that one-shot command. The sleep is to give the release a chance to take effect before attempting the one-shot command. Without it, we could easily be sending the viewer a command to release a restriction and the command to issue the one-shot in the same frame and I wouldn't want to assume that the viewer will process these in the correct order in that case.